### ISSUE TITLE
validation error: dict=type1FontDict required entry=FirstChar missing

### ISSUE THREAD

**Issue by simon-franz** [pdfcpu/pdfcpu#258]
HI there,
I've a pdf-file which was created by Docutain Version 1.58 (PDF Version 1.6).
Opening the file in Chrome and Acrobat Reader does work without problem.
Trying to validate the file results in an validation error:
`validating(mode=relaxed) dm.pdf ...
validation error: dict=type1FontDict required entry=FirstChar missing`

The detailed logging is:
```
validating(mode=relaxed) .\dm.pdf ...
 READ: 2020/12/03 10:56:54 Read: begin
 INFO: 2020/12/03 10:56:54 PDF Version 1.5 conforming reader
 READ: 2020/12/03 10:56:54 readXRefTable: begin
 READ: 2020/12/03 10:56:54 scanning for offsetLastXRefSection starting at 368538
 READ: 2020/12/03 10:56:54 Offset last xrefsection: 368810
 READ: 2020/12/03 10:56:54 buildXRefTableStartingAt: begin
 READ: 2020/12/03 10:56:54 headerVersion begin
 READ: 2020/12/03 10:56:54 headerVersion: end, found header version: 1.6
 READ: 2020/12/03 10:56:54 newPositionedReader: positioned to offset: 368810
 READ: 2020/12/03 10:56:54 line: <xref>
 READ: 2020/12/03 10:56:54 buildXRefTableStartingAt: found xref section
 READ: 2020/12/03 10:56:54 parseXRefSection begin
 READ: 2020/12/03 10:56:54 parseXRefSection: <0 8>
 READ: 2020/12/03 10:56:54 parseXRefTableSubSection: begin
 READ: 2020/12/03 10:56:54 detected xref subsection, startObj=0 length=8
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: begin
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Object #0 is unused, next free is object#0, generation=65535
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Insert new xreftable entry for Object 0
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: end
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: begin
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Object #1 is in use at offset=10, generation=0
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Insert new xreftable entry for Object 1
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: end
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: begin
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Object #2 is in use at offset=61, generation=0
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Insert new xreftable entry for Object 2
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: end
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: begin
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Object #3 is in use at offset=121, generation=0
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Insert new xreftable entry for Object 3
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: end
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: begin
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Object #4 is in use at offset=227, generation=0
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Insert new xreftable entry for Object 4
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: end
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: begin
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Object #5 is in use at offset=425, generation=0
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Insert new xreftable entry for Object 5
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: end
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: begin
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Object #6 is in use at offset=11751, generation=0
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Insert new xreftable entry for Object 6
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: end
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: begin
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Object #7 is in use at offset=368530, generation=0
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: Insert new xreftable entry for Object 7
 READ: 2020/12/03 10:56:54 parseXRefTableEntry: end
 READ: 2020/12/03 10:56:54 parseXRefTableSubSection: end
 READ: 2020/12/03 10:56:54 parseXRefSection: All subsections read!
 READ: 2020/12/03 10:56:54 parseXRefSection: parsing trailer dict..
 READ: 2020/12/03 10:56:54 line (len 7) <trailer>
 READ: 2020/12/03 10:56:54 line: <>
 READ: 2020/12/03 10:56:54 line: <<</Root 1 0 R /Info 7 0 R /Size 8>>>
 READ: 2020/12/03 10:56:54 scanTrailer dictBuf after start tag: <<</Root 1 0 R /Info 7 0 R /Size 8>>>
 READ: 2020/12/03 10:56:54 processTrailer: trailerString: (len:36) <<</Root 1 0 R /Info 7 0 R /Size 8>> >
 READ: 2020/12/03 10:56:54 processTrailer: trailerDict:
<<
	<Info, (7 0 R)>
	<Root, (1 0 R)>
	<Size, 8>
>>
 READ: 2020/12/03 10:56:54 parseTrailerDict begin
 READ: 2020/12/03 10:56:54 parseTrailerInfo begin
 READ: 2020/12/03 10:56:54 parseTrailerInfo: Root object: (1 0 R)
 READ: 2020/12/03 10:56:54 parseTrailerInfo: Info object: (7 0 R)
 READ: 2020/12/03 10:56:54 parseTrailerInfo end
 READ: 2020/12/03 10:56:54 parseTrailerDict end
 READ: 2020/12/03 10:56:54 buildXRefTableStartingAt: end
TRACE: 2020/12/03 10:56:54 EnsureValidFreeList begin
TRACE: 2020/12/03 10:56:54 EnsureValidFreeList: empty free list.
 READ: 2020/12/03 10:56:54 readXRefTable: end
 READ: 2020/12/03 10:56:54 dereferenceXRefTable: begin
 READ: 2020/12/03 10:56:54 decodeObjectStreams: begin
 READ: 2020/12/03 10:56:54 decodeObjectStreams: end
 READ: 2020/12/03 10:56:54 dereferenceObjects: begin
 READ: 2020/12/03 10:56:54 dereferenceObject: begin, dereferencing object 0
 READ: 2020/12/03 10:56:54 free object 0
 READ: 2020/12/03 10:56:54 dereferenceObject: begin, dereferencing object 1
 READ: 2020/12/03 10:56:54 in use object 1
 READ: 2020/12/03 10:56:54 dereferenceObject: dereferencing object 1
 READ: 2020/12/03 10:56:54 ParseObject: begin, obj#1, offset:10
 READ: 2020/12/03 10:56:54 newPositionedReader: positioned to offset: 10
 READ: 2020/12/03 10:56:54 object: small obj w/o stream, parse until endobj
 READ: 2020/12/03 10:56:54 dict: end, #1
 READ: 2020/12/03 10:56:54 dereferenceObject: end obj 1 of 8
<<<
	<Pages, (2 0 R)>
	<Type, Catalog>
>>>
 READ: 2020/12/03 10:56:54 logStream: no ObjectStreamDict
 READ: 2020/12/03 10:56:54 dereferenceObject: begin, dereferencing object 2
 READ: 2020/12/03 10:56:54 in use object 2
 READ: 2020/12/03 10:56:54 dereferenceObject: dereferencing object 2
 READ: 2020/12/03 10:56:54 ParseObject: begin, obj#2, offset:61
 READ: 2020/12/03 10:56:54 newPositionedReader: positioned to offset: 61
 READ: 2020/12/03 10:56:54 object: small obj w/o stream, parse until endobj
 READ: 2020/12/03 10:56:54 dict: end, #2
 READ: 2020/12/03 10:56:54 dereferenceObject: end obj 2 of 8
<<<
	<Count, 1>
	<Kids, [(4 0 R)]>
	<Type, Pages>
>>>
 READ: 2020/12/03 10:56:54 logStream: no ObjectStreamDict
 READ: 2020/12/03 10:56:54 dereferenceObject: begin, dereferencing object 3
 READ: 2020/12/03 10:56:54 in use object 3
 READ: 2020/12/03 10:56:54 dereferenceObject: dereferencing object 3
 READ: 2020/12/03 10:56:54 ParseObject: begin, obj#3, offset:121
 READ: 2020/12/03 10:56:54 newPositionedReader: positioned to offset: 121
 READ: 2020/12/03 10:56:54 object: small obj w/o stream, parse until endobj
 READ: 2020/12/03 10:56:54 dict: end, #3
 READ: 2020/12/03 10:56:54 dereferenceObject: end obj 3 of 8
<<<
	<BaseFont, Arial>
	<Encoding, WinAnsiEncoding>
	<Name, F1>
	<Subtype, Type1>
	<Type, Font>
>>>
 READ: 2020/12/03 10:56:54 logStream: no ObjectStreamDict
 READ: 2020/12/03 10:56:54 dereferenceObject: begin, dereferencing object 4
 READ: 2020/12/03 10:56:54 in use object 4
 READ: 2020/12/03 10:56:54 dereferenceObject: dereferencing object 4
 READ: 2020/12/03 10:56:54 ParseObject: begin, obj#4, offset:227
 READ: 2020/12/03 10:56:54 newPositionedReader: positioned to offset: 227
 READ: 2020/12/03 10:56:54 object: small obj w/o stream, parse until endobj
 READ: 2020/12/03 10:56:54 dict: end, #4
 READ: 2020/12/03 10:56:54 dereferenceObject: end obj 4 of 8
<<<
	<Contents, (5 0 R)>
	<MediaBox, [0 0 881 1175]>
	<Parent, (2 0 R)>
	<Resources, <<
		<Font, <<
			<F1, (3 0 R)>
		>>>
		<ProcSet, [PDF Text ImageB ImageC ImageI]>
		<XObject, <<
			<I6, (6 0 R)>
		>>>
	>>>
	<Type, Page>
>>>
 READ: 2020/12/03 10:56:54 logStream: no ObjectStreamDict
 READ: 2020/12/03 10:56:54 dereferenceObject: begin, dereferencing object 5
 READ: 2020/12/03 10:56:54 in use object 5
 READ: 2020/12/03 10:56:54 dereferenceObject: dereferencing object 5
 READ: 2020/12/03 10:56:54 ParseObject: begin, obj#5, offset:425
 READ: 2020/12/03 10:56:54 newPositionedReader: positioned to offset: 425
 READ: 2020/12/03 10:56:54 buffer: endInd=-1 streamInd=28
 READ: 2020/12/03 10:56:54 object: big stream, we parse object until stream
 READ: 2020/12/03 10:56:54 pdfFilterPipeline: begin
 READ: 2020/12/03 10:56:54 streamDictForObject: end, Streamobject #5
 READ: 2020/12/03 10:56:54 LoadEncodedStreamContent: begin
<<
	<Length, 11269>
>>
 READ: 2020/12/03 10:56:54 newPositionedReader: positioned to offset: 461
 READ: 2020/12/03 10:56:54 LoadEncodedStreamContent: seeked to offset:461
 READ: 2020/12/03 10:56:54 readContentStream: begin streamLength:11269
 READ: 2020/12/03 10:56:54 readContentStream: count=11269, buflen=11269(2C05)
 READ: 2020/12/03 10:56:54 readContentStream: end
 READ: 2020/12/03 10:56:54 LoadEncodedStreamContent: end: len(streamDictRaw)=11269
 READ: 2020/12/03 10:56:54 saveDecodedStreamContent: begin decode=false
 READ: 2020/12/03 10:56:54 dereferenceObject: end obj 5 of 8
<<<
	<Length, 11269>
>>>
 READ: 2020/12/03 10:56:54 logStream: no stream content
 READ: 2020/12/03 10:56:54 dereferenceObject: begin, dereferencing object 6
 READ: 2020/12/03 10:56:54 in use object 6
 READ: 2020/12/03 10:56:54 dereferenceObject: dereferencing object 6
 READ: 2020/12/03 10:56:54 ParseObject: begin, obj#6, offset:11751
 READ: 2020/12/03 10:56:54 newPositionedReader: positioned to offset: 11751
 READ: 2020/12/03 10:56:54 buffer: endInd=-1 streamInd=155
 READ: 2020/12/03 10:56:54 object: big stream, we parse object until stream
 READ: 2020/12/03 10:56:54 pdfFilterPipeline: begin
 READ: 2020/12/03 10:56:54 pdfFilterPipeline: end w/o decode parms
 READ: 2020/12/03 10:56:54 streamDictForObject: end, Streamobject #6
 READ: 2020/12/03 10:56:54 LoadEncodedStreamContent: begin
<<
	<BitsPerComponent, 8>
	<ColorSpace, DeviceRGB>
	<Filter, DCTDecode>
	<Height, 3264>
	<Length, 356595>
	<Subtype, Image>
	<Type, XObject>
	<Width, 2448>
>>
 READ: 2020/12/03 10:56:54 newPositionedReader: positioned to offset: 11914
 READ: 2020/12/03 10:56:54 LoadEncodedStreamContent: seeked to offset:11914
 READ: 2020/12/03 10:56:54 readContentStream: begin streamLength:356595
 READ: 2020/12/03 10:56:54 readContentStream: count=356595, buflen=356595(570F3)
 READ: 2020/12/03 10:56:54 readContentStream: end
 READ: 2020/12/03 10:56:54 LoadEncodedStreamContent: end: len(streamDictRaw)=356595
 READ: 2020/12/03 10:56:54 saveDecodedStreamContent: begin decode=false
 READ: 2020/12/03 10:56:54 dereferenceObject: end obj 6 of 8
<<<
	<BitsPerComponent, 8>
	<ColorSpace, DeviceRGB>
	<Filter, DCTDecode>
	<Height, 3264>
	<Length, 356595>
	<Subtype, Image>
	<Type, XObject>
	<Width, 2448>
>>>
 READ: 2020/12/03 10:56:54 logStream: no stream content
 READ: 2020/12/03 10:56:54 dereferenceObject: begin, dereferencing object 7
 READ: 2020/12/03 10:56:54 in use object 7
 READ: 2020/12/03 10:56:54 dereferenceObject: dereferencing object 7
 READ: 2020/12/03 10:56:54 ParseObject: begin, obj#7, offset:368530
 READ: 2020/12/03 10:56:54 newPositionedReader: positioned to offset: 368530
 READ: 2020/12/03 10:56:54 object: small object w/o stream, parse until endobj
 READ: 2020/12/03 10:56:54 dict: end, #7
 READ: 2020/12/03 10:56:54 dereferenceObject: end obj 7 of 8
<<<
	<Author, (INFOSOFT Informations - und Dokumentationssysteme GmbH)>
	<Bezeichnung, (Docutain Dokument)>
	<CreationDate, (D:20201126133225+01'00')>
	<Creator, (Docutain® Version 1.58)>
	<ModDate, (D:20201126133225+01'00')>
	<Producer, (Docutain® Version 1.58)>
	<Title, (Docutain Dokument)>
>>>
 READ: 2020/12/03 10:56:54 logStream: no ObjectStreamDict
 READ: 2020/12/03 10:56:54 dereferenceObjects: end
 READ: 2020/12/03 10:56:54 identifyRootVersion: begin
 READ: 2020/12/03 10:56:54 dereferenceXRefTable: end
 READ: 2020/12/03 10:56:54 Read: end
 INFO: 2020/12/03 10:56:54 validating
VALID: 2020/12/03 10:56:54 *** validateXRefTable begin ***
VALID: 2020/12/03 10:56:54 *** validateRootObject begin ***
VALID: 2020/12/03 10:56:54 validateNameEntry begin: entry=Type
VALID: 2020/12/03 10:56:54 validateNameEntry end: entry=Type
VALID: 2020/12/03 10:56:54 validateResources: This page node has 1 pages
VALID: 2020/12/03 10:56:54 validateRectangleEntry begin: entry=MediaBox
VALID: 2020/12/03 10:56:54 validateNumberArrayEntry begin: entry=MediaBox
VALID: 2020/12/03 10:56:54 validateArrayEntry begin: entry=MediaBox
VALID: 2020/12/03 10:56:54 validateRectangleEntry begin: entry=CropBox
VALID: 2020/12/03 10:56:54 validateNumberArrayEntry begin: entry=CropBox
VALID: 2020/12/03 10:56:54 validateArrayEntry begin: entry=CropBox
VALID: 2020/12/03 10:56:54 validateIntegerEntry begin: entry=Rotate
VALID: 2020/12/03 10:56:54 validatePagesDict: PageNode: (4 0 R)
VALID: 2020/12/03 10:56:54 validateNameEntry begin: entry=BaseFont
VALID: 2020/12/03 10:56:54 validateNameEntry end: entry=BaseFont
VALID: 2020/12/03 10:56:54 validateIntegerEntry begin: entry=FirstChar
TRACE: 2020/12/03 10:56:54 freeList begin
STATS: 2020/12/03 10:56:54 XRefTable:
*************************************************************************************************
HeaderVersion: 1.6
has 1 pages
XRefTable:
                     Size: 8
              Root object: (1 0 R)
              Info object: (7 0 R)
XRefTable with 8 entres:
    0: f   next=       0 generation=65535
    1:   offset=      10 generation=0 pdfcpu.Dict type=Catalog 
<<
	<Pages, (2 0 R)>
	<Type, Catalog>
>>
    2:   offset=      61 generation=0 pdfcpu.Dict type=Pages 
<<
	<Count, 1>
	<Kids, [(4 0 R)]>
	<Type, Pages>
>>
    3:   offset=     121 generation=0 pdfcpu.Dict type=Font subType=Type1 
<<
	<BaseFont, Arial>
	<Encoding, WinAnsiEncoding>
	<Name, F1>
	<Subtype, Type1>
	<Type, Font>
>>
    4:   offset=     227 generation=0 pdfcpu.Dict type=Page 
<<
	<Contents, (5 0 R)>
	<MediaBox, [0 0 881 1175]>
	<Parent, (2 0 R)>
	<Resources, <<
		<Font, <<
			<F1, (3 0 R)>
		>>>
		<ProcSet, [PDF Text ImageB ImageC ImageI]>
		<XObject, <<
			<I6, (6 0 R)>
		>>>
	>>>
	<Type, Page>
>>
    5:   offset=     425 generation=0 pdfcpu.StreamDict 
<<
	<Length, 11269>
>>
decoded stream content (length = 0)
<>
    6:   offset=   11751 generation=0 pdfcpu.StreamDict 
<<
	<BitsPerComponent, 8>
	<ColorSpace, DeviceRGB>
	<Filter, DCTDecode>
	<Height, 3264>
	<Length, 356595>
	<Subtype, Image>
	<Type, XObject>
	<Width, 2448>
>>
decoded stream content (length = 0)
<>
    7:   offset=  368530 generation=0 pdfcpu.Dict 
<<
	<Author, (INFOSOFT Informations - und Dokumentationssysteme GmbH)>
	<Bezeichnung, (Docutain Dokument)>
	<CreationDate, (D:20201126133225+01'00')>
	<Creator, (Docutain® Version 1.58)>
	<ModDate, (D:20201126133225+01'00')>
	<Producer, (Docutain® Version 1.58)>
	<Title, (Docutain Dokument)>
>>

Empty free list.

Total pages: 1
No font info available.

No image info available.


STATS: 2020/12/03 10:56:54 Timing:
STATS: 2020/12/03 10:56:54 read                 :  0.049s  76.5%
STATS: 2020/12/03 10:56:54 validate             :  0.015s  23.5%
STATS: 2020/12/03 10:56:54 total processing time:  0.064s

STATS: 2020/12/03 10:56:54 Original:
STATS: 2020/12/03 10:56:54 File size            : 360 KB (369050 bytes)
STATS: 2020/12/03 10:56:54 Total binary data    : 359 KB (367864 bytes) 99.7%
STATS: 2020/12/03 10:56:54 Total other data     : 1 KB (1186 bytes)  0.3%

Fatal: dict=type1FontDict required entry=FirstChar missing
github.com/pdfcpu/pdfcpu/pkg/pdfcpu.Dict.Entry
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/pdfcpu/dict.go:113
github.com/pdfcpu/pdfcpu/pkg/pdfcpu/validate.validateIntegerEntry
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/pdfcpu/validate/objects.go:521
github.com/pdfcpu/pdfcpu/pkg/pdfcpu/validate.validateType1FontDict
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/pdfcpu/validate/font.go:677
github.com/pdfcpu/pdfcpu/pkg/pdfcpu/validate.validateFontDict
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/pdfcpu/validate/font.go:967
github.com/pdfcpu/pdfcpu/pkg/pdfcpu/validate.validateFontResourceDict
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/pdfcpu/validate/font.go:1000
github.com/pdfcpu/pdfcpu/pkg/pdfcpu/validate.validateResourceDict
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/pdfcpu/validate/pages.go:45
github.com/pdfcpu/pdfcpu/pkg/pdfcpu/validate.validatePageResources
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/pdfcpu/validate/pages.go:114
github.com/pdfcpu/pdfcpu/pkg/pdfcpu/validate.validatePageDict
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/pdfcpu/validate/pages.go:762
github.com/pdfcpu/pdfcpu/pkg/pdfcpu/validate.validatePagesDict
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/pdfcpu/validate/pages.go:966
github.com/pdfcpu/pdfcpu/pkg/pdfcpu/validate.validatePages
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/pdfcpu/validate/pages.go:1005
github.com/pdfcpu/pdfcpu/pkg/pdfcpu/validate.validateRootObject
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/pdfcpu/validate/xReftable.go:856
github.com/pdfcpu/pdfcpu/pkg/pdfcpu/validate.XRefTable
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/pdfcpu/validate/xReftable.go:33
github.com/pdfcpu/pdfcpu/pkg/api.ValidateContext
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/api/api.go:70
github.com/pdfcpu/pdfcpu/pkg/api.Validate
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/api/validate.go:50
github.com/pdfcpu/pdfcpu/pkg/api.ValidateFile
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/api/validate.go:91
github.com/pdfcpu/pdfcpu/pkg/cli.Validate
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/cli/cli.go:33
github.com/pdfcpu/pdfcpu/pkg/cli.Process
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/cli/process.go:35
main.process
	C:/Users/Simon/Documents/GitHub/pdfcpu/cmd/pdfcpu/process.go:88
main.processValidateCommand
	C:/Users/Simon/Documents/GitHub/pdfcpu/cmd/pdfcpu/process.go:126
main.commandMap.process
	C:/Users/Simon/Documents/GitHub/pdfcpu/cmd/pdfcpu/cmd.go:145
main.main
	C:/Users/Simon/Documents/GitHub/pdfcpu/cmd/pdfcpu/main.go:55
runtime.main
	c:/go/src/runtime/proc.go:204
runtime.goexit
	c:/go/src/runtime/asm_amd64.s:1374
validation error
github.com/pdfcpu/pdfcpu/pkg/api.Validate
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/api/validate.go:55
github.com/pdfcpu/pdfcpu/pkg/api.ValidateFile
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/api/validate.go:91
github.com/pdfcpu/pdfcpu/pkg/cli.Validate
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/cli/cli.go:33
github.com/pdfcpu/pdfcpu/pkg/cli.Process
	C:/Users/Simon/Documents/GitHub/pdfcpu/pkg/cli/process.go:35
main.process
	C:/Users/Simon/Documents/GitHub/pdfcpu/cmd/pdfcpu/process.go:88
main.processValidateCommand
	C:/Users/Simon/Documents/GitHub/pdfcpu/cmd/pdfcpu/process.go:126
main.commandMap.process
	C:/Users/Simon/Documents/GitHub/pdfcpu/cmd/pdfcpu/cmd.go:145
main.main
	C:/Users/Simon/Documents/GitHub/pdfcpu/cmd/pdfcpu/main.go:55
runtime.main
	c:/go/src/runtime/proc.go:204
runtime.goexit
	c:/go/src/runtime/asm_amd64.s:1374
```

If you want i can share the pdf via email.
Thanks in advance!

**Comment by hhrutter** [comment]
This is a potential spec violation:
The PDF spec says "FirstChar" is required since V1.5 except for standard 14 fonts which "Arial" is not.
I don't know if relaxing the spec is the right thing to do in this case.

**Comment by hhrutter** [comment]
Please send me a sample if you can so I can provide a fix.
Thank you!

**Comment by maxiride** [comment]
I've encountered this same error recently and sadly had to try to circumvent it by using ValidationNone as a configuration option.

I noticed however that this check is bypassed only by using ValidationRelaxed, like if ValidationNone still enforced this validation.

Behaviour:
- validate with ValidationStrict: error raised
- validate with ValidationRelaxed: error silenced
- validate with ValidationNone: error raised -> expected behaviour: error silenced


It is my understanding that the validations have a precedence on each other, so I expect ValidationNone to also silence the error. Am I understanding the validation wrongly? Always happy to learn.
